src/igvfd/schemas/mixins.json

```diff
@@ -290,5 +290,18 @@
             "comment": "See source.json for available identifiers.",
             "type": "string"
         }
+    },
+    "sample_material": {
```

Comment:
@KAndreeva ,  I am not sure that it matters, but why did you choose to put this in mixins to be available to all objects versus in technical sample only to be used by technical_samples only?

Reply:
I thought this might apply to future derived classes from Sample at later date. If you think this will never occur, I can move it to being a property of technical sample only?

Reply:
This change for sample material to not be in mixins is complete.